### PR TITLE
80 hide the buttons in the tables

### PR DESCRIPTION
**Pull request by Eleni001**

### CODE REVIEW

File: app/components/admin/AdminDashboard.tsx [hunk 1]
```diff
@@ -158,7 +163,44 @@ export default function AdminDashboard({ products: newProducts }: Props) {
...
+            })}
+          </Tbody>
+        </Table>
+      </TableContainer>
+    </>
+  );
+}
+
+{
+  /* <Flex justify="center" alignItems="center" height="100%">
```
Comment (ipieropan) [inline app/components/admin/AdminDashboard.tsx:203]: u could remove the commented code if we are not using it

File: app/components/admin/OrderCard.tsx [hunk 2]
```diff
@@ -1,18 +1,24 @@
...
+import { BsThreeDots } from 'react-icons/bs';
 import OrderSentButton from './OrderSentButton';
 
 interface Props {
   order: OrderWithInformation;
 }
 
 export default function OrderCard(props: Props) {
-  const [loading, setLoading] = useState(false);
+  /* const [loading, setLoading] = useState(false);
```
Comment (ipieropan) [inline app/components/admin/OrderCard.tsx:21]: this commented code aswell

**Review by ipieropan — CHANGES_REQUESTED** [review]

File: app/components/admin/AdminDashboard.tsx (hunk 1 above)
Comment (Eleni001) [reply app/components/admin/AdminDashboard.tsx]: thanks you! I have removed them now

**Review by ipieropan — APPROVED** [review]

**Review by Flowball — APPROVED** [review]
bien

**Pull request merged**